Kubernetes/legos/k8s_get_memory_utilization_of_services/k8s_get_memory_utilization_of_services.py

```diff
@@ -34,14 +34,16 @@ def k8s_get_memory_utilization_of_services_printer(output):
     if status:
         print("All services are within memory utilization threhsold")
     else:
-        headers = ["Pod", "Namespace", "Utilization %"]
+        headers = ["Service", "Pod", "Namespace","Container","Utilization %"]
         table_data = []
 
         for entry in data:
+            service= entry['service']
             pod = entry['pod']
             namespace = entry['namespace']
+            container = entry['container_name']
             utilization_percentage = entry.get('utilization_percentage', "")
-            table_data.append([pod, namespace, utilization_percentage])
+            table_data.append([service, pod, namespace, container, utilization_percentage])
```

Comment:
`None` handling is missing - what if the pod is suspended and hence the `utilization_percentage` is not available.

Also, is `container_name` always set? Should it be:

```
container = entry.get('container_name', "NA")
```

and similarly for other keys.

Reply:
Done